src/ChatServer/ChatServer.cs

```diff
@@ -292,12 +292,14 @@ private async ValueTask ChatClientAcceptingAsync(CancelEventArgs e)
         e.Cancel = this.CurrentConnections >= this.Settings.MaximumConnections;
     }
 
-    private async ValueTask ChatClientAcceptedAsync(ClientAcceptedEventArgs e)
```

Comment:
it's async by purpose, please revert

Reply:
What's the reason? 

The reason for changes is "This async method lacks 'await' operators and will run synchronously"

Reply:
First, it doesn't hurt... Second, exception handling might be better - I can look that up or think about that, but I'm too tired right now 😅